CONTRIBUTING.md

```diff
@@ -10,6 +10,33 @@ Ideally name your branches with prefixes and descriptions, like this: `[type]/[c
 - `try/` = experimental feature, "tentatively add"
 - `update/` = update an existing feature
 
-For example, `add/foo-module` means you're working on adding a new foo module. 
+For example, `add/foo-module` means you're working on adding a new foo module.
 
 You can pick among all the <a href="https://github.com/WordPress/packages/issues">tickets</a>, or some of the ones labelled <a href="https://github.com/WordPress/packages/labels/Good%20First%20Task">Good First Task</a>.
+
+
+## Releasing
+
+This repository uses [lerna](https://lernajs.io) to manage and release the packages. Lerna automatically releases all the outdated packages. To check which packages are outdated and will be released, type `lerna updated`.
```

Comment:
This assumes `lerna` is installed globally on the developer's computer. Assuming npm 5.x+, we could recommend `npx lerna updated`.

Reply:
Added a PR #58